src/Parser/Warlock/Affliction/Modules/WarlockCore/Channeling.js

```diff
@@ -13,15 +13,15 @@ import { formatMilliseconds } from 'common/format';
  */
 class Channeling extends CoreChanneling {
   on_byPlayer_cast(event) {
-    if (event.ability.guid === SPELLS.DRAIN_SOUL.id) {
+    if (event.ability.guid === SPELLS.DRAIN_SOUL_TALENT.id) {
```

Comment:
Since Drain Life is its own ability now, I'd suggest also adding handling for that (other specs might need that too).

Reply:
Could probably make it a shared Warlock module, and have Affliction one extend the shared one, adding Drain Soul as well. I'll add it as TODO for the future since I am little confused by the code.